Recrutify/Recrutify.DataAccess/Repositories/Abstract/IUserRepository.cs

```diff
@@ -1,10 +1,13 @@
-﻿using System.Threading.Tasks;
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
 using Recrutify.DataAccess.Models;
 
 namespace Recrutify.DataAccess.Repositories.Abstract
 {
     public interface IUserRepository : IBaseRepository<User>
     {
         Task<User> GetByEmailAsync(string email);
+
+        public Task<List<User>> GetByRolesAsync();
```

Comment:
pass roles in List parameter - repository should not contain any business logic, it doesn't knows which roles you want